examples/connext_dds/group_coherent_presentation/py/GroupCoherentExampleType.py

```diff
@@ -0,0 +1,52 @@
+#
+# (c) 2022 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software solely for use with RTI products.  The Software is
+# provided "as is", with no warranty of any type, including any warranty for
+# fitness for any purpose. RTI is under no obligation to maintain or support
+# the Software.  RTI shall not be liable for any incidental or consequential
+# damages arising out of the use or inability to use the software.
+#
+
+import rti.idl as idl
+from enum import IntEnum, auto
+from dataclasses import field
+
+
+@idl.enum
+class AlarmCode(IntEnum):
+    PATIENT_OK = auto()
+    PATIENT_REQUEST = auto()
+    SENSOR_OFFLINE = auto()
+    ABNORMAL_READING = auto()
+
+
+@idl.struct(
+    type_annotations={
```

Comment:
```suggestion
    member_annotations={
```

Did you generate this with rtiddsgen?

Reply:
I did not, I'll generate it from rtiddsgen so that all the type_annotiations are changed to member_annotations and instead of int, it will specify idl.int32